examples/gluon_alexnet/gluon_alexnet.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import mxnet
+from mxnet import gluon
+from mxnet.gluon import nn
+from mms.model_service.mxnet_vision_service import MXNetVisionService
+import numpy as np
+
+"""
+MMS examples for loading Gluon Hybrid/Dynamic models
+"""
+
+
+class ImperativeAlexNet(gluon.Block):
+    """
+    Fully imperative gluon Alexnet model
+    """
+    def __init__(self, classes=1000, **kwargs):
+        super(ImperativeAlexNet, self).__init__(**kwargs)
+        with self.name_scope():
+            self.features = nn.Sequential(prefix='')
+            with self.features.name_scope():
+                self.features.add(nn.Conv2D(64, kernel_size=11, strides=4,
+                                            padding=2, activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Conv2D(192, kernel_size=5, padding=2,
+                                            activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Conv2D(384, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.Conv2D(256, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.Conv2D(256, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Flatten())
+                self.features.add(nn.Dense(4096, activation='relu'))
+                self.features.add(nn.Dropout(0.5))
+                self.features.add(nn.Dense(4096, activation='relu'))
+                self.features.add(nn.Dropout(0.5))
+            self.output = nn.Dense(classes)
+
+    def forward(self, F, x):
+        x = self.features(x)
+        x = self.output(x)
+        return x
+
+
+class HybridAlexNet(gluon.HybridBlock):
```

Comment:
You are using the same network for two different examples: Hybrid and non hybrid. I think this is confusing to users who may not understand why AlexNet is an example to non hybrid, where it can be hybridized. Why not use a different example, in a separate file, for non hybrid?

Reply:
Done. Split this file into three examples "imperative", "hybrid" and "pretrained"